### PR TITLE
created loader component

### LINKED ISSUE #6: Add loader while calling API.

### PR DESCRIPTION
**Pull request by ardnahcivar**
Fixes #6
-
showing the loader component, when fetching data through api

### CODE REVIEW

**Comment by ardnahcivar** [comment]
@arshadkazmi42 which spinner should we use , how about MoonLoader or FadeLoader

**Comment by arshadkazmi42** [comment]
> @arshadkazmi42 which spinner should we use , how about MoonLoader or FadeLoader

Both looks good. You can take the call on this

**Comment by ardnahcivar** [comment]
@arshadkazmi42 requested changes are done

**Comment by arshadkazmi42** [comment]
@akramkazmi71 Can you have a final look at this. From my end its looks good

File: src/App.js [hunk 1]
```diff
@@ -79,16 +87,23 @@ class App extends Component{
...
         <div class="container">
-         {this.state.repoName}
-          {this.state.pageNumber>1 && <button class="nextPrev" onClick={this.previousPage}>Previous Page</button>}
-          <button class="nextPrev" onClick={this.nextPage}>Next Page</button> 
+          {this.state.loader ?
+            <Loader loader={this.state.loader}/>
+            :
+            this.state.repoName
+          }
+          {this.state.pageNumber>1 && <button onClick={this.previousPage}>Previous Page</button>}
```
Comment (akramkazmi71) [inline src/App.js]: Can you mention the class="nextPrev" in the button tag?
Other than that everything looks good :100:

Comment (ardnahcivar) [reply src/App.js]: you mean on both the buttons

Comment (akramkazmi71) [reply src/App.js]: Yes, both the buttons, but due to #15 , there is a change in code. So update it as per the changes in mentioned PR.

**Comment by akramkazmi71** [comment]
Can you resolve the conflicts occurred due to #15 .

File: src/App.js (hunk 1 above)
Comment (ardnahcivar) [reply src/App.js]: @akramkazmi71  done the changes

**Comment by ardnahcivar** [comment]
> Can you resolve the conflicts occurred due to #15 .

done

**Pull request merged**